cli/create.go

```diff
@@ -60,6 +60,10 @@ func (cc *CreateCommand) runCreate(args []string) error {
 
 	ctx := context.Background()
 	apiClient := cc.cli.Client()
+	if err := pullMissingImage(ctx, apiClient, config.Image, false); err != nil {
```

Comment:
This functionality is just implemented in the CLI side, not the API server side?
I am wondering if this is reasonable? @Ace-Tang @fuweid 
Just double check this.

Reply:
The best will be done in api server, hope someone can do that.

Reply:
@allencloud @Ace-Tang I don't think it should be done in the API server, because we don't know the client want to pull the missing image and it will make the API heavy. Currently, the `Pull` API allows the caller to do the customize like `pullMissingImage`. It's good enough, I think.

Reply:
As you said, @fuweid , one api just do one thing, seems make sense.